topology/University of Wisconsin/GLOW ATLAS/WISC-ATLAS.yaml

```diff
@@ -24,61 +24,6 @@ Resources:
           uri_override: higgs08.cs.wisc.edu:2119
     VOOwnership:
       ATLAS: 100
-  WISC-ATLAS-SE:
-    Active: true
```

Comment:
Hm. Maybe given https://github.com/opensciencegrid/topology/pull/3#pullrequestreview-113374949 I should have kept these and done `Active: false` and `Disable: true`?

Reply:
We don't use "Disable" any more, it's essentially vestigial.

These hosts haven't sent in any recent (<2 years) records so it's OK to delete them; otherwise we would just set "Active" to false and do a pruning pass later.

Reply:
Great and thanks! :+1: